ebusd-2.1.x/de/vaillant/15.emm.csv

```diff
@@ -0,0 +1,99 @@
+# type (r[1-9];w;u),circuit,name,[comment],[QQ],ZZ,PBSB,[ID],field1,part (m/s),datatypes/templates,divider/values,unit,comment,field2,part (m/s),datatypes/templates,divider/values,unit,comment,field3,part (m/s),datatypes/templates,divider/values,unit,comment,field4,part (m/s),datatypes/templates,divider/values,unit,comment
+#,EMM00,sensoDIREKT VRC710,,,,,,,,,,,
+
+*[SW],scan,,,SW,,,,,,,,,
+
+*r,,,,,08,"B524","02000000",,,IGN:4,,,
```

Comment:
all of the messages have 08 set as target, to basically the config is for 08 (a heater) and not 15 (a controller).
so something is totally awkward with that.
please share the scan result and some example hex sequences to clarify this

Reply:
I'll try to clarify.
Yes, all messages have 08 as target. From my observations the 15 device (EMM0, VRC710) changes the behavior / unlocks certain behavior of the 08 device. So the messages are only available, when EMM0 and BAI are online. So what is needed is a mechanism to define a message that is dependent on versions of more than one circuit / device. In this case 08 and 15.
This is why I decided to define message in a 15 configuration file with 08 as target.

Reply:
But to be honest, I'm not sure if really all messages are only available if both circuits are online.

Reply:
hm interesting. please share your scan result.
this is really weird and I have not seen such before, so this needs to be sorted out carefully

Reply:
scan id = no data stored
scan.05  = no data stored
scan.08  = Vaillant;BAI00;1003;7603
scan.08 id = 21;23;23;0010043897;0001;009317;N9
scan.15  = Vaillant;EMM00;0104;8503
scan.15 id = 21;23;24;0020274790;0082;027273;N1
scan.f6  = Vaillant;NETX2;4035;5703
scan.f6 id = 21;23;32;0020260962;0933;127538;N5